### PR TITLE
burpsuite: version bump.

### PR DESCRIPTION
**Pull request by wykwit**
burpsuite version bump and url update so that it points to the latest release

### CODE REVIEW

File: packages/burpsuite/PKGBUILD [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
-pkgver=2021.5.3
+pkgver=2021.6.2
 pkgrel=1
 epoch=1
 groups=('blackarch' 'blackarch-webapp' 'blackarch-proxy' 'blackarch-scanner'
         'blackarch-fuzzer')
 pkgdesc='An integrated platform for attacking web applications (community edition) + SHELLING plugin.'
 arch=('any')
 url='https://portswigger.net/burp/releases/professional-community-2021-2-1?requestededition=community'
+url='https://portswigger.net/burp/releases/community/latest'
```
Comment (ikstream) [inline packages/burpsuite/PKGBUILD:13]: Why do you add a `url` instead of updating the old one?

Please update `url` instead, as it is outdated anyways

**Review by ikstream — CHANGES_REQUESTED** [review]

File: packages/burpsuite/PKGBUILD (hunk 1 above)
Comment (wykwit) [reply packages/burpsuite/PKGBUILD:12]: whoops, overlooked it by mistake

**Comment by noptrix** [comment]
@wykwit thank you. we updated it internally. btw, you don't have to touch `source`. simply `pkgver` is enough as long as the url to `source` didn't change. thanks again. closing here

**Comment by wykwit** [comment]
@noptrix, there's still one more thing - the `url` update. Could you commit that as well?

**Pull request closed without merging**